scripts/a11y-sla.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+set -e
+
+        echo "🚀 Fetching open issues from GitHub API (handling pagination)..."
+
+        # Define SLA Thresholds based on impact levels (weeks)
+        declare -A LABEL_THRESHOLDS=( ["Blocker"]=4 ["Critical"]=10 ["Serious"]=20 ["Moderate"]=30 )
+
+        # SLA Labels
+        SLA_LABELS=("SLA P1" "SLA P2" "SLA P3" "SLA Breach")
+
+        # Required labels for filtering
+        REQUIRED_LABELS=("A11y" "VPAT")
+        LABELS_QUERY=$(IFS=,; echo "${REQUIRED_LABELS[*]}")  # Convert to "A11y,VPAT"
+
+        PAGE=1
+        ALL_ISSUES="[]"
+        CURRENT_TIMESTAMP=$(date -u +%s)
+
+        # Fetch issues with pagination
+        while :; do
+          echo "🔄 Fetching page $PAGE..."
+          RESPONSE=$(curl -s -H "Authorization: Bearer $GITHUB_TOKEN" \
+            -H "Accept: application/vnd.github.v3+json" \
+            "https://api.github.com/repos/$GITHUB_REPOSITORY/issues?state=open&labels=$LABELS_QUERY&per_page=100&page=$PAGE")
+
+          # Break if no more issues
+          if [[ "$(echo "$RESPONSE" | jq '. | length')" -eq 0 ]]; then
+            echo "✅ No more issues to fetch. Exiting loop."
+            break
+          fi
+
+          # Merge response
+          ALL_ISSUES=$(echo "$ALL_ISSUES $RESPONSE" | jq -s 'add')
+
+          ((PAGE++))
+        done
+
+        # Get issue count
+        TOTAL_ISSUES=$(echo "$ALL_ISSUES" | jq '. | length')
+        echo "📊 Total Issues Fetched: $TOTAL_ISSUES"
+
+        if [[ "$TOTAL_ISSUES" -eq 0 ]]; then
+          echo "⚠️ No issues found with the required labels."
+          exit 0
+        fi
+
+        declare -A GROUPED_ISSUES
+
+        # Process issues
+        mapfile -t ISSUES < <(echo "$ALL_ISSUES" | jq -c '.[]')
+
+        for issue in "${ISSUES[@]}"; do
+          CREATED_AT=$(echo "$issue" | jq -r '.created_at')
+          ISSUE_NUMBER=$(echo "$issue" | jq -r '.number')
+
+          # Convert date to readable format
+          CREATED_DATE=$(date -d "$CREATED_AT" +"%Y-%m-%d" 2>/dev/null || date -u -jf "%Y-%m-%dT%H:%M:%SZ" "$CREATED_AT" +"%Y-%m-%d")
+
+          # Calculate issue age in weeks
+          CREATED_TIMESTAMP=$(date -d "$CREATED_AT" +%s 2>/dev/null || date -u -jf "%Y-%m-%dT%H:%M:%SZ" "$CREATED_AT" +%s)
+          DAYS_OLD=$(( (CURRENT_TIMESTAMP - CREATED_TIMESTAMP) / 86400 ))
+          WEEKS_OLD=$(( DAYS_OLD / 7 ))
+
+          # Fetch issue labels
+          LABELS=$(echo "$issue" | jq -r '[.labels[].name] | join(", ")')
+
+          # Determine impact level
+          IMPACT_LEVEL=""
+          for level in "${!LABEL_THRESHOLDS[@]}"; do
+            if [[ "$LABELS" == *"$level"* ]]; then
```

Comment:
Do we need/want the `*`? If we do any kind of loose checking, it seems like it should be to not be case sensitive. Something like:

```suggestion
           if [[ "${LABELS,,}" == "${level,,}" ]]; then
```

Reply:
Provided suggestion is matching if LABELS exactly equals Blocker, Critical, etc, But GitHub issues usually have multiple labels, and LABELS is like "A11y, VPAT, Serious", so LABELS != Serious exactly. so i changed to `if echo "$LABELS" | grep -iq "$level"; then`